lambda-buffers-frontend/resources/good/work-dir/compiler-input.textproto

```diff
@@ -0,0 +1,979 @@
+modules {
```

Comment:
I'm keeping them around because I need something to point at. In the following work, I'll structure these as goldens and use them in tests (currently these 2 are not used in tests)